outputs.tf

```diff
@@ -61,3 +65,36 @@ resource "local_file" "deploy_openstack" {
   filename        = "ansible/files/deploy-openstack.sh"
   file_permission = "0755"
 }
+
+resource "ansible_host" "control_host" {
```

Comment:
The module docs show how to define variables such as ansible_user, which would avoid specifying it manually: https://nbering.github.io/terraform-provider-ansible/docs/resources/ansible_host.html

Reply:
Added `ansible_user` to `ansible_group.cluster_group`.